### PR TITLE
Add daf yomi to hdate

### PR DESCRIPTION
**Pull request by moshekaplan**

### CODE REVIEW

**Comment by moshekaplan** [comment]
@tsvi : Could you please review this PR?

**Comment by tsvi** [comment]
If you could fix your PR, I'll try to get an updated version of hdate up by motsei shabbes.

**Comment by tsvi** [comment]
This fixes part of #17 by the way. 😊

File: tests/test_hdate.py [hunk 1]
```diff
@@ -397,6 +397,16 @@ def test_get_omer_day(self, execution_number, rand_hdate):
             rand_hdate.hdate = HebrewDate(rand_hdate.hdate.year, 9, day)
             assert rand_hdate.omer_day == day + 44
 
+    def test_daf_yomi(self):
+        # Random test date
+        myhdate = HDate(gdate=datetime.date(2014, 4, 28), hebrew=False)
+        assert myhdate.daf_yomi == ("Beitzah", 29)
```
Comment (tsvi) [inline tests/test_hdate.py]: Please add a test for hebrew as well

File: hdate/date.py [hunk 2]
```diff
@@ -235,6 +235,23 @@ def omer_day(self):
...
+    @property
+    def daf_yomi(self):
+        """Return a tuple of mesechta and daf."""
+        days_since_start_cycle_11 = (self.gdate - htables.DAF_YOMI_CYCLE_11_START).days
+        daf_number = days_since_start_cycle_11 % (htables.DAF_YOMI_TOTAL_PAGES)
+        for mesechta in htables.DAF_YOMI_MESECHTOS:
+            if daf_number >= mesechta["pages"]:
+                daf_number -= mesechta["pages"]
+            else:
+                break
```
Comment (tsvi) [inline hdate/date.py]: Add +2 to daf number over here

File: hdate/date.py [hunk 3]
```diff
@@ -235,6 +235,23 @@ def omer_day(self):
...
+        if self.hebrew:
+            heb_number = hebrew_number(daf_number + 2)
+            heb_number = heb_number.replace("'", "").replace('"', "")
+            return mesechta["heb_name"], heb_number
+        else:
+            return mesechta["en_name"], daf_number + 2
```
Comment (tsvi) [inline hdate/date.py]: Maybe return daf number as a string so we don't have a discrepancy on the type between English and Hebrew.

Also no need for else as we already returned. A nicer way might be:

    if self.hebrew:
        daf =
        mesechta =
    else:
        ...
    return mesechta, daf

File: hdate/htables.py [hunk 4]
```diff
@@ -814,3 +815,48 @@ class HolidayTypes(Enum):
...
+    {"en_name": "Zevachim", "heb_name": "זבחים", "pages": 119},
+    {"en_name": "Menachos", "heb_name": "מנחות", "pages": 109},
+    {"en_name": "Chullin", "heb_name": "חולין", "pages": 141},
+    {"en_name": "Bechoros", "heb_name": "בכורות", "pages": 60},
+    {"en_name": "Arachin", "heb_name": "ערכין", "pages": 33},
+    {"en_name": "Temurah", "heb_name": "תמורה", "pages": 33},
+    {"en_name": "Kereisos", "heb_name": "כריתות", "pages": 27},
+    {"en_name": "Meilah", "heb_name": "מעילה", "pages": 36},
+    {"en_name": "Niddah", "heb_name": "נדה", "pages": 72},
+]
```
[the remark this answers is not in the document]
Comment (moshekaplan) [reply hdate/htables.py]: Fixed in a6d7f7bce7455c7c608e35e2ebebf52238082df3

File: hdate/date.py [hunk 5]
```diff
@@ -235,6 +235,23 @@ def omer_day(self):
...
+        days_since_start_cycle_11 = (self.gdate - htables.DAF_YOMI_CYCLE_11_START).days
+        daf_number = days_since_start_cycle_11 % (htables.DAF_YOMI_TOTAL_PAGES)
+        for mesechta in htables.DAF_YOMI_MESECHTOS:
+            if daf_number >= mesechta["pages"]:
+                daf_number -= mesechta["pages"]
+            else:
+                break
+        if self.hebrew:
+            heb_number = hebrew_number(daf_number + 2)
+            heb_number = heb_number.replace("'", "").replace('"', "")
```
[the remark this answers is not in the document]
Comment (moshekaplan) [reply hdate/date.py]: Fixed in a6d7f7bce7455c7c608e35e2ebebf52238082df3

File: tests/test_hdate.py (hunk 1 above)
Comment (moshekaplan) [reply tests/test_hdate.py]: Fixed in a6d7f7bce7455c7c608e35e2ebebf52238082df3

File: hdate/date.py (hunk 2 above)
Comment (moshekaplan) [reply hdate/date.py]: Please look at what I did in a6d7f7bce7455c7c608e35e2ebebf52238082df3 , I think it's clearer that way

File: hdate/date.py (hunk 3 above)
Comment (moshekaplan) [reply hdate/date.py]: Fixed in a6d7f7bce7455c7c608e35e2ebebf52238082df3

**Pull request merged**